scripts/kconfig/kconfigfunctions.py

```diff
@@ -76,8 +76,18 @@ def dt_hex_val(kconf, _, name, unit=None):
 
     return hex(d)
 
+def dt_str_val(kconf, _, name):
+    """
+    This function looks up 'name' in the DTS generated "conf" style database
+    and if its found it will return the value as string.
+    """
+    if doc_mode or name not in dt_defines:
+        return ""
+
+    return dt_defines[name].strip('\"')
```

Comment:
Can use `"'"` and `'"'` to represent a string with just `'` or `"` as well.

This might be problematic if `"` ever appears legit at the start/end of a value too, but maybe that'll never happen.